lib/project_types/script/layers/infrastructure/languages/assemblyscript_project_creator.rb

```diff
@@ -11,23 +11,73 @@ class AssemblyScriptProjectCreator
           property! :script_name, accepts: String
           property! :path_to_project, accepts: String
 
-          BOOTSTRAP = "npx --no-install shopify-scripts-toolchain-as bootstrap --from %{extension_point} --dest %{base}"
           BUILD = "shopify-scripts-toolchain-as build --src src/shopify_main.ts " \
           "--binary build/script.wasm --metadata build/metadata.json"
-          MIN_NODE_VERSION = "14.5.0"
+          MIN_NODE_VERSION = "14.5.0" # kept because task_runner uses this
```

Comment:
oh yeah 🤔 maybe the task runner could check the script's package.json file to see if theres a specified `engines.node` version